csrc/scheduler/reduction.cpp

```diff
@@ -77,6 +77,25 @@ std::shared_ptr<ReductionParams> innerReductionHeuristic(
 
   const int64_t n_elems = total_reduction_numel * total_iteration_numel;
 
+  if (n_elems == 0) {
```

Comment:
Instead of having this condition in both `innerReductionHeuristic` and `outerReductionHeuristic`, can we have just one in `reductionHeuristic`?

Reply:
Does this still happen? Don't the pointwise and noop schedulers take care of all these conditions?

Reply:
I have the same question.